### ISSUE TITLE
Implement convac in torch instead of NumPy

### ISSUE THREAD

**Issue by idfah** [idfah/cebl#13]
*Description*

The current NumPy-only implementation of `convac` / CNN-LA is somewhat difficult to maintain because of manual gradient computation.  It also does not support GPUs.  This story is to re-implement this network architecture in `pytorch` with optional GPU training support.

*Acceptance Criteria*

- [ ] New implementation of `convac` available and written in `pytorch`
- [ ] New `convac` implementation supports both CPU and GPU training

*Tasks*

- [ ] create new module structure for torch re-implementations
- [ ] implement convac
- [ ] add GPU support
- [ ] add unit tests
